backend/databasemodel/model.sql

```diff
@@ -1486,8 +1501,11 @@ CREATE TABLE kooste.tamperewfs_luontopolkureitit (
 	name text NOT NULL DEFAULT 'Nimitieto puuttuu',
 	tarmo_category text DEFAULT 'Ulkoilureitit',
 	type_name text DEFAULT 'Luontopolku',
+	deleted boolean NOT NULL DEFAULT false,
 	CONSTRAINT tamperewfs_luontopolkureitit_pk PRIMARY KEY (tunnus)
 );
+CREATE INDEX ON kooste.tamperewfs_luontopolkureitit (deleted);
+CREATE INDEX ON kooste.tamperewfs_luonnonmuistomerkit (visibility);
```

Comment:
I think there is wrong table name here (kooste.tamperewfs_luonnonmuistomerkit -> kooste.tamperewfs_luontopolkureitit) :)